robot/basestation/app.py

```diff
@@ -94,7 +106,8 @@ def ping_rover():
     if error:
         print("Error: " + error.decode())
 
-    ros_output, error = run_shell("rosrun ping_acknowledgment ping_response_client.py hello")
+    ros_output, error = run_shell(
+        "rosrun ping_acknowledgment ping_response_client.py hello")
```

Comment:
We've changed how this works so you don't need to send something. If we wanna pass something, we can pass "api" and it will return the ping value instead of human readable nice text

```suggestion 
        "rosrun ping_acknowledgment ping_response_client.py")
```